book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Great job so far @c00p75 you're making good progress. 🥳  I wanted to provide some feedback on the implementation of the `many-to-many relationship` between `Person` and `Book` using the intermediate class `Rental`.

- [ ] In your Book class, you've correctly defined the rentals array to store the associated rentals. However, to fully meet the requirements, you need to implement the logic to associate Rental instances with Book objects. 💯 

- [ ] I recommend adding a method called `add_rental` in your `Book class `that allows you to associate rentals with a book 👍 By implementing this logic, you'll fully establish the `has-many relationship` between `Book` and `Rental`, fulfilling the requirements of the project 💯 
- Project Requirement

> Create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. The following should be implemented:
Create the has-many side of Book and Rental (a book has many rentals).